cmd/olympus/actions/goget.go

```diff
@@ -30,7 +30,8 @@ func goGetMeta(c buffalo.Context, getter cdnmetadata.Getter) error {
 	}
 	loc, err := getter.Get(params.Module)
 	if err != nil {
-		return c.Error(http.StatusNotFound, fmt.Errorf("module %s does not exist", params.Module))
+		msg := fmt.Sprintf("module %s does not exist", params.Module)
```

Comment:
just thinking about reusing ErrModuleVersionNotFound from storage package and returning that.